common/rpc/interceptor/namespace_count_limit.go

```diff
@@ -81,6 +82,16 @@ func (ni *NamespaceCountLimitInterceptor) Intercept(
 	_, methodName := SplitMethodName(info.FullMethod)
 	// token will default to 0
 	token := ni.tokens[methodName]
+
+	if token != 0 {
+		// for GetWorkflowExecutionHistoryRequest, we only care about long poll requests
+		longPollReq, ok := req.(*workflowservice.GetWorkflowExecutionHistoryRequest)
+		if ok && !longPollReq.WaitNewEvent {
```

Comment:
All five methods now have independent counters.. should we collapse the three new ones into one, so we have  {pollworkflow, pollactivity, otherlongpolls}?